hw/xbox/nv2a/nv2a.c

```diff
@@ -34,20 +33,18 @@ int ffs(register int valu)
 {
     register int bit;
 
-    if (valu == 0)
+    if (valu == 0) {
```

Comment:
Doesn't `ERROR: use ctz32() instead of ffs()` from PR description mean we can remove this function entirely?

If not, while we are at it: `value`?

Reply:
Agree to change `valu` to `value`.

Think s/`ffs`/`ctz32`/ should happen in another PR.

Reply:
Sounds good to me.
I'll create an issue about this one so we don't forget about it.

Reply:
This was not done yet. But it's fine with me, given that it will be removed in the future anyway.